cppForSwig/Makefile

```diff
@@ -40,23 +42,33 @@ else
 PYTHON_INCLUDES=$(shell $(PYVER)-config --includes )
 endif
 
-CPPFLAGS += $(ARMORY_CPPFLAGS) -Icryptopp -Imdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
-LDLIBS += -lpthread -Lmdb
-SWIG_OPTS    += -c++ -python -threads
-
-SWIG_INC     += 
-
-CXXCPP += $(CPPFLAGS) $(PYTHON_INCLUDES) -std=c++11
+# "-Wl,--no-as-needed" used for Ubuntu 12.04 due to rt library linking issues.
+# http://stackoverflow.com/questions/17150075/undefined-reference-to-clock-gettime-although-lrt-is-given
+# has more info. Require only for static linking. When 12.04 goes, remove this
+# unless it's discovered that OS X needs it.
+ifdef STATIC_LINK
+LDLIBS += -Wl,--no-as-needed
```

Comment:
@josephbisch - Just pushed a modified version of the patch. (I also forgot to credit you again. Sorry! For now, perhaps you should push patches both here and to 0.93-bugfix.) Your basic idea works. My only mod was to force the code to compile only under Linux. Workarounds should be limited only to affected operating systems. Otherwise, this is good. :) Ugly, but I really don't see a better alternative. I don't want --no-as-needed to cause binary bloat down the line; we're considering making standalone binaries, a la Bitcoin Core. Bloat will be a big issue in that case.

The 64-bit comment was something mentioned to me last night. Eventually, I believe the idea is to separate the code between the offline-only & online-only pieces. I don't know if that would have an effect here. It was just something mentioned to me.